src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="html">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-wed calendar-month-length--29">
+      <div class="calendar__day calendar__day--day1"></div>
```

Comment:
remove seconds class for all days
to better understand the idea - check the comment in css
```suggestion
      <div class="calendar__day"></div>
```

Reply:
All done!